plugins/rapid7_insightvm/help.md

```diff
@@ -38,6 +38,45 @@ Example input:
 
 ### Actions
 
+#### Add User Asset Group Access
+
+This action is used to grant an user account access to an asset group by ID.
```

Comment:
```suggestion
This action is used to grant a user account access to an asset group by ID.
```
This should ideally be corrected in the plugin.spec.yaml so the regeneration sticks - but, I can't find this action in the plugin.spec.yaml 🤔